modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/shibari_stand.dm

```diff
@@ -0,0 +1,314 @@
+		if(HAS_TRAIT(current_mob, TRAIT_ROPEBUNNY))
+			current_mob.handcuffed.breakouttime = 4 MINUTES
+		current_mob.update_abstract_handcuffed()
+
+//Restore the position of the mob after unbuckling.
+/obj/structure/chair/shibari_stand/post_unbuckle_mob(mob/living/buckled)
+	buckled.pixel_x = buckled.base_pixel_x + buckled.body_position_pixel_x_offset
+	buckled.pixel_y = buckled.base_pixel_y + buckled.body_position_pixel_y_offset - 4
+	buckled.layer = initial(buckled.layer)
+
+	cut_overlay(shibari_shadow_overlay)
+	cut_overlay(shibari_rope_overlay)
+	cut_overlay(shibari_rope_overlay_behind)
+
+	if(current_mob)
+		if(current_mob.handcuffed)
+			current_mob.handcuffed.dropped(current_mob)
+		current_mob.set_handcuffed(null)
+		current_mob.update_abstract_handcuffed()
+
+	if(ropee)
+		ropee.forceMove(get_turf(src))
+	current_mob = null
+
+/obj/item/restraints/handcuffs/milker/shibari
+	name = "ropes"
+	desc = "A shibari rope for restraining hands."
+	breakouttime = 2 MINUTES
+
+/obj/item/shibari_stand_kit
+	name = "shibari stand construction kit"
+	desc = "Construction requires a wrench."
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_structures/bdsm_furniture.dmi'
+	throwforce = 0
+	icon_state = "shibari_kit"
+	w_class = WEIGHT_CLASS_HUGE
+	var/current_color = "pink"
+	var/static/list/shibarikit_designs
+
+/obj/item/shibari_stand_kit/proc/populate_shibarikit_designs()
+	shibarikit_designs = list(
+		"pink" = image (icon = src.icon, icon_state = "shibari_kit_pink"),
+		"teal" = image (icon = src.icon, icon_state = "shibari_kit_teal"),
+		"black" = image (icon = src.icon, icon_state = "shibari_kit_black"),
+		"red" = image (icon = src.icon, icon_state = "shibari_kit_red"),
+		"white" = image (icon = src.icon, icon_state = "shibari_kit_white"))
+
+// Default initialization
+/obj/item/shibari_stand_kit/Initialize()
+	. = ..()
+	populate_shibarikit_designs()
+	//random color variation on start. Because why not?
+	current_color = pick(shibarikit_designs)
+	update_icon_state()
+	update_icon()
+
+/obj/item/shibari_stand_kit/update_icon_state()
+	. = ..()
+	icon_state = "[initial(icon_state)]_[current_color]"
+
+//Changing examine for this item
+/obj/item/shibari_stand_kit/examine(mob/user)
+	.=..()
+	. += span_notice("Looks like it can be customized with a <b>screwdriver</b>.")
+
+//to change model
+/obj/item/shibari_stand_kit/screwdriver_act(mob/living/user, obj/item/tool)
+	var/choice = show_radial_menu(user,src, shibarikit_designs, custom_check = CALLBACK(src, .proc/check_menu, user, tool), radius = 36, require_near = TRUE)
+	if(!choice)
+		return FALSE
+	if(tool.use_tool(src, user, 0, volume=40))
+		to_chat(user, span_notice("You switched the frame's plastic fittings color to [choice]."))
+		current_color = choice
+		update_appearance()
+	return TRUE
+
+/obj/item/shibari_stand_kit/proc/check_menu(mob/living/user)
+	if(!istype(user))
+		return FALSE
+	if(user.incapacitated())
+		return FALSE
+	return TRUE
```

Comment:
There needs to be more checks here, like a range check so you can't do this from literally across the map.